docs/integrations/microsoft-azure/azure-functions.md

```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
+
+### Network
+
+The **Azure Functions - Network** dashboard provides details on network traffic and connectivity related to your Azure Functions. This includes data on inbound and outbound traffic, network latency, and potential network issues.
+
+Use this dashboard to:
+* Track network activity and performance of your Azure Functions.
+* Diagnose network-related issues and connectivity problems.
+* Monitor network usage and optimize network configuration.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Network.png')} alt="Azure Functions Network dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Operations
+
+The **Azure Functions - Operations** dashboard provides an overview of operational metrics and activities related to your Azure Functions. This includes I/O Read, Write bytes/second and I/O Read, Write operations/second.
+
+Use this dashboard to:
+* Monitor the operational status and health of your Azure Functions.
+* Review deployment activities and execution metrics.
+* Analyze operational trends and optimize function management.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Operations.png')} alt="Azure Functions Operations dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### OS Statistics
+
+The **Azure Functions - OS Statistics** dashboard provides information on operating system metrics and statistics relevant to your Azure Functions. This includes details on current assemblies, number of processor threads and other OS-level performance indicators.
+
+Use this dashboard to:
+* Track OS-level performance metrics impacting your Azure Functions.
+* Monitor Current assemblies, threads, and handles for optimal performance.
+* Identify and address OS-related performance issues.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-OS-Statistics.png')} alt="Azure Functions OS Statistics dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Performance
+
+The **Azure Functions - Performance** dashboard provides insights into the performance of your Azure Functions. This includes metrics on execution times, throughput, and overall function efficiency.
+
+Use this dashboard to:
+* Monitor and analyze the performance of your Azure Functions.
+* Identify performance bottlenecks and optimize function execution.
+* Review performance metrics and trends to ensure optimal function performance.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Performance.png')} alt="Azure Functions Performance dashboard" style={{border: '1px solid gray'}} width="800" />
```

Comment:
Crop the extra space in the dashboard image.

Reply:
Done